### DISCUSSION TITLE
I delete an organization and can't recreate it

### DISCUSSION THREAD

**Discussion by muffinpowered** [community/community#113732] [New to GitHub]
I want to register a company "Quick Studio" at https://github.com/studioquick, but nothing comes out on this or any other account.....

![browser_m0Trb6Uw1s](https://github.com/community/community/assets/104167738/92e97145-fbd5-4207-ae4f-6ded0ea61fad)

**Comment by vuvanthuan** [comment]
You can wait a few hours and search again and check your organization's privacy settings which have been included about account verification. Hope that can help you
